index.js

```diff
@@ -1,30 +1,37 @@
 import React from 'react';
 import malarkey from 'malarkey';
 
-const MalarkeyComponent = React.createClass({
-  getDefaultProps () {
-    return {
-      messages: [],
-      options: {
-        loop: true
-      },
-      sequence (m, messages) {
-        messages.forEach(message => {
-          m.type(message).pause().delete();
-        });
-      }
-    };
-  },
+export default class Malarkey extends React.Component {
+  constructor (props) {
+    super(props);
+  }
 
   componentDidMount () {
     const {messages, options, sequence} = this.props;
     const m = malarkey(this.node, options);
     sequence(m, messages);
-  },
+  }
 
   render () {
-    return <div ref={node => this.node = node} />;
+    const {messages, staticRender} = this.props;
+    if (staticRender) {
+      // return a random message on static rendering
+      return <span>{ messages[Math.floor(Math.random() * messages.length)] }</span>;
+    }
+    return <span ref={node => this.node = node} />;
   }
-});
+}
 
-export default MalarkeyComponent;
+Malarkey.displayName = 'Malarkey';
+Malarkey.defaultProps = {
```

Comment:
Does not seem to work with our setup. I suspect it might work with Babel 6, but the upgrade is still a WIP for us.